.travis/smoke-test.sh

```diff
@@ -9,5 +9,6 @@ function finish {
 trap finish EXIT
 
 docker-compose --file docker-compose.yaml up -d web
+.scripts/docker/wait-healthy.sh browser_app_1
 docker-compose --file docker-compose.yaml exec app bin/console --version
 nc -z localhost 8080
```

Comment:
Technically wants `browser_web_1` too. Maybe we need to be able to wait on multiple containers?

Reply:
By default `nginx` doesn't define an healthcheck:
```
$ docker-compose ps
    Name                   Command                 State              Ports        
-----------------------------------------------------------------------------------
browser_app_1   docker-php-entrypoint php-fpm   Up (healthy)   9000/tcp            
browser_web_1   nginx -g daemon off;            Up             0.0.0.0:8080->80/tcp
```
We could add one in `docker-compose.yaml` and wait for both, though I've never seen such a problem with `nginx` (more on the underlying apps or databases)

Reply:
Probably not worth the hassle.